packages/core/integration-tests/test/server.js

```diff
@@ -152,7 +152,7 @@ describe('server', function() {
     subscription = await b.watch();
     await getNextBuild(b);
 
-    await sleep(10000);
+    await sleep(4000);
```

Comment:
can this be removed entirely? not sure why I didn't before since awaiting `getNextBuild` should take care of this

Reply:
looks like it :)